### DISCUSSION TITLE
An output error when coupling modflow with cwatm

### DISCUSSION THREAD

**Discussion by hsk0059** [iiasa/CWatM#137] [Q&A]
Hello, Dear professors, 

I am a reseracher in NUS Singapore, and I have been studying CWatM-Modflow recently, but now I have a trouble in outputing the stoGroundwater variable. Therefore, **I have two questions here**.

**Q1:** I focus on GWSA and I know GWSA = storGroundwater - unmet when limitAbstraction = False. When I just run the CWatm model, it can successfully output the stoGroundwater variable. But if I coupled modflow with it, evenif I define "OUT_Map_MonthEnd = stoGroundwater", it will report an error that the variable "stoGroundwater" is not defined after the warming period. If I change to "OUT_Map_MonthEnd = tws" (or other variables), the CWatM-Modflow model can run successfully again. I donot know how to resolve it.  

**Q2:** Is there other ways to get the GWSA information when successfully outputing all variables except stoGroundwater? I notice there is a file called groundwater_storage_available_monthavg.nc, what does it represent? In another way, I can also get modflow_watertable.nc, can I use it to indirectly get GWSA?

Looking forward to your reply.

**Comment by mikhailsmilovic** [comment]
Guten Tag @hsk0059,

There are different output variables for groundwater depending on if one is using standard CWatM or with the modflow coupling. All the following units are in metres.

## Standard CWatM:
storGroundwater = groundwater storage
sum_capRiseFromGW = capillary rise
nonFossilGroundwaterAbs = abstractions from storGroundwater
unmetDemand = groundwater abstractions beyond storGroundwater

## With Modflow coupling:
groundwater_storage_available = groundwater storage
capillary = capillary rise
Pumping_daily = pumping in CWatM (then sent to Modflow)
modfPumpingM_actual = pumping in CWatM (returned from Modflow)

## Both:
baseflow
